requirements/prod.txt

```diff
@@ -11,4 +11,6 @@ psycopg2==2.8.5
 python3-openid==3.2.0
 geoip2==4.0.1
 django-ipware==3.0.0
-django-apscheduler==0.3.1
+django-apscheduler==0.4.0
+git+https://github.com/Chive/django-multiupload.git#egg=django-multiupload
```

Comment:
We need to use the github version because that has support for Django >= 2.2

We can do two things here:
-  Consider this is done (not merge) and wait for the [release on PyPI](https://github.com/Chive/django-multiupload/issues/42)
- Merge now, open an issue to fix this later when it's released.

Reply:
I'd go with the second option honestly, I think its better if we merge it for now and not keep this PR open. What do you prefer more?

Reply:
I also prefer the second option because I do not know when the package will be released or whether it will be released at all.